RecoCTPPS/ProtonReconstruction/src/ProtonReconstructionAlgorithm.cc

```diff
@@ -130,8 +130,8 @@ double ProtonReconstructionAlgorithm::ChiSquareCalculator::operator()(const doub
     const double y = k_out.y - oit->second.y0;
 
     // calculate chi^2 contributions, convert track data mm --> cm
-    const double x_diff_norm = (x - track->getX() * 1E-1) / (track->getXUnc() * 1E-1);
-    const double y_diff_norm = (y - track->getY() * 1E-1) / (track->getYUnc() * 1E-1);
+    const double x_diff_norm = (x - track->x() * 1E-1) / (track->yUnc() * 1E-1);
```

Comment:
```xUnc()```

Reply:
Thank you very much @perrotta! Fixed in 
9c02087